mne/datasets/hf_sef/hf_sef.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# Authors: Jussi Nurminen <jnu@iki.fi>
+# License: BSD Style.
+
+
+import tarfile
+import os.path as op
+import os
+from ...utils import _fetch_file, verbose
+from ..utils import _get_path, logger, _do_path_update
+
+
+@verbose
+def data_path(set='evoked', path=None, force_update=False, update_path=None,
```

Comment:
I just noticed this -- don't use `set` as a var name it shadows a builtin

Reply:
Indeed..